sites/partners/src/listings/PaperListingDetails/sections/DetailAdditionalFees.tsx

```diff
@@ -5,6 +5,20 @@ import { getDetailFieldString } from "./helpers"
 
 const DetailAdditionalFees = () => {
   const listing = useContext(ListingContext)
+  const getUtilitiesIncluded = () => {
+    let utilitiesExist = false
+    const utilities = Object.keys(listing?.utilities ?? {}).map((utility) => {
+      if (listing?.utilities[utility]) {
+        utilitiesExist = true
+        return (
+          <li className={"list-disc mx-5 mb-1 md:w-1/3 w-full grow"}>
+            {t(`listings.utilities.${utility}`)}
+          </li>
+        )
+      }
+    })
+    return utilitiesExist ? utilities : <>None</>
```

Comment:
We can use the `t.none` string here (looks existing in the features piece as well)